From 840d6b6420e1fd8cdf6e4de7fa58a6f8de151622 Mon Sep 17 00:00:00 2001
From: Yann Dirson <yann@blade-group.com>
Date: Tue, 6 Apr 2021 17:28:45 +0200
Subject: [PATCH] Set serial console baudrate back to 1500000.
Upstream-Status: Inappropriate[other]

TF-A runs between two u-boot stages which both uses 1500000 baud, it
just makes no sense to use the same UART at a different rate.

This effectively reverts part of 0c05748bdebfad9fa43a80962186438bb8fbce62.
Main reason for that change stated in https://developer.trustedfirmware.org/T762
is ChromeOS compatibility.

Looks like this patch may become unnecessary in the future, when
u-boot and TF-A get to communicate this value.

---
 plat/rockchip/rk3399/rk3399_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/rockchip/rk3399/rk3399_def.h b/plat/rockchip/rk3399/rk3399_def.h
index ba83242eb..8d6ecfbe6 100644
--- a/plat/rockchip/rk3399/rk3399_def.h
+++ b/plat/rockchip/rk3399/rk3399_def.h
@@ -17,7 +17,7 @@
 /**************************************************************************
  * UART related constants
  **************************************************************************/
-#define RK3399_BAUDRATE			115200
+#define RK3399_BAUDRATE			1500000
 #define RK3399_UART_CLOCK		24000000
 
 /******************************************************************************
-- 
2.30.2

